Formula/tezos-signer-http.rb

```diff
@@ -0,0 +1,83 @@
+# SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+class TezosSignerHttp < Formula
+  url "file:///dev/null"
+  version "v8.2-3"
+
+  bottle :unneeded
+  depends_on "tezos-signer"
+
+  desc "Meta formula that provides backround tezos-signer service that runs over http"
+
+  def install
+    startup_contents =
+      <<~EOS
+      #!/usr/bin/env bash
+
+      set -euo pipefail
+
+      signer="/usr/local/bin/tezos-signer"
+
+      if [[ -n $PIDFILE ]]; then
+        pid_file_args=("--pid-file" "$PIDFILE")
+      else
+        pid_file_args=()
+      fi
+
+      if [[ -n $MAGIC_BYTES ]]; then
+        magic_bytes_args=("--magic-bytes" "$MAGIC_BYTES")
+      else
+        magic_bytes_args=()
+      fi
+
+      if [[ -n $CHECK_HIGH_WATERMARK ]]; then
+        check_high_watermark_args=("--check-high-watermark")
+      else
+        check_high_watermark_args=()
+      fi
+
+      "$signer" -d "$DATA_DIR" launch http signer --address "$ADDRESS" --port "$PORT" \
+        ${pid_file_args[@]+"${pid_file_args[@]}"} ${magic_bytes_args[@]+"${magic_bytes_args[@]}"} \
+        ${check_high_watermark_args[@]+"${check_high_watermark_args[@]}"} "$@"
+    EOS
+    File.write("tezos-signer-http-start", startup_contents)
+    bin.install "tezos-signer-http-start"
+  end
```

Comment:
Have you intentionally omitted something like this here `plist_options manual: "tezos-node-mainnet start"`

Reply:
yep, because this service isn't supposed to be run manually, one should use `tezos-signer` formula instead